go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/nspcc-dev/rfc6979
 
-go 1.21
+go 1.22
```

Comment:
why do you close the issue with _this_ commit?

Reply:
Because it updates to go 1.22.

Reply:
the issue has a reference to another issue which is a checklist with some requirements for updating